### ISSUE TITLE
Giada fails to compile

### ISSUE THREAD

**Issue by blablack** [monocasual/giada#52]
Giada's current git (27th of October) fails to compile.

Here is the error I get:

```
g++ -DPACKAGE_NAME=\"giada\" -DPACKAGE_TARNAME=\"giada\" -DPACKAGE_VERSION=\"0.10\" -DPACKAGE_STRING=\"giada\ 0.10\" -DPACKAGE_BUGREPORT=\"giadaloopmachine@gmail.com\" -DPACKAGE_URL=\"\" -DPACKAGE=\"giada\" -DVERSION=\"0.10\" -DHAVE_LIBPTHREAD=1 -DSTDC_HEADERS=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_MEMORY_H=1 -DHAVE_STRINGS_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_UNISTD_H=1 -I.    -Wno-parentheses -std=c++0x -O2 -pipe -mtune=generic -msse -msse2 -mfpmath=sse -fstack-protector --param ssp-buffer-size=4 -D_FORTIFY_SOURCE=2 -fPIC -DPIC  -c -o tests/giada_tests-main.o `test -f 'tests/main.cpp' || echo './'`tests/main.cpp
g++ -DPACKAGE_NAME=\"giada\" -DPACKAGE_TARNAME=\"giada\" -DPACKAGE_VERSION=\"0.10\" -DPACKAGE_STRING=\"giada\ 0.10\" -DPACKAGE_BUGREPORT=\"giadaloopmachine@gmail.com\" -DPACKAGE_URL=\"\" -DPACKAGE=\"giada\" -DVERSION=\"0.10\" -DHAVE_LIBPTHREAD=1 -DSTDC_HEADERS=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_MEMORY_H=1 -DHAVE_STRINGS_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_UNISTD_H=1 -I.    -Wno-parentheses -std=c++0x -O2 -pipe -mtune=generic -msse -msse2 -mfpmath=sse -fstack-protector --param ssp-buffer-size=4 -D_FORTIFY_SOURCE=2 -fPIC -DPIC  -c -o tests/giada_tests-conf.o `test -f 'tests/conf.cpp' || echo './'`tests/conf.cpp
g++ -DPACKAGE_NAME=\"giada\" -DPACKAGE_TARNAME=\"giada\" -DPACKAGE_VERSION=\"0.10\" -DPACKAGE_STRING=\"giada\ 0.10\" -DPACKAGE_BUGREPORT=\"giadaloopmachine@gmail.com\" -DPACKAGE_URL=\"\" -DPACKAGE=\"giada\" -DVERSION=\"0.10\" -DHAVE_LIBPTHREAD=1 -DSTDC_HEADERS=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_MEMORY_H=1 -DHAVE_STRINGS_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_UNISTD_H=1 -I.    -Wno-parentheses -std=c++0x -O2 -pipe -mtune=generic -msse -msse2 -mfpmath=sse -fstack-protector --param ssp-buffer-size=4 -D_FORTIFY_SOURCE=2 -fPIC -DPIC  -c -o tests/giada_tests-patch.o `test -f 'tests/patch.cpp' || echo './'`tests/patch.cpp
tests/patch.cpp: In function 'void ____C_A_T_C_H____T_E_S_T____6()':
tests/patch.cpp:109:11: error: 'class Patch' has no member named 'masterInPlugins'
     patch.masterInPlugins.add(plugin1);
           ^
tests/patch.cpp:109:31: error: 'plugin1' was not declared in this scope
     patch.masterInPlugins.add(plugin1);
                               ^
tests/patch.cpp:110:11: error: 'class Patch' has no member named 'masterOutPlugins'
     patch.masterOutPlugins.add(plugin2);
           ^
tests/patch.cpp:110:32: error: 'plugin2' was not declared in this scope
     patch.masterOutPlugins.add(plugin2);
                                ^
Makefile:2072: recipe for target 'tests/giada_tests-patch.o' failed
```

**Comment by monocasual** [comment]
Hi @blablack, is this still an issue with the latest commit(s)?

**Comment by blablack** [comment]
@monocasual it still happens.

It fails when packaging or running
```
make giada_tests
```
the packaging system seems to build tests automatically.

**Comment by monocasual** [comment]
Missing #ifdefs! Should be fixed with debf651 . Thanks!

**Comment by blablack** [comment]
Confirmed - issue is fixed.
